plasma_framework/test/src/utils/Quarantine.test.js

```diff
@@ -0,0 +1,67 @@
+const Quarantine = artifacts.require('Quarantine');
+const QuarantineMock = artifacts.require('QuarantineMock');
+const { expect } = require('chai');
+const { constants, expectRevert, time } = require('openzeppelin-test-helpers');
+
+contract('Quarantine', () => {
+    before('setup', async () => {
+        const quarantine = await Quarantine.new();
+        await QuarantineMock.link('Quarantine', quarantine.address);
+    });
+
+    describe('contract is quarantined', () => {
+        const QUARANTINE_PERIOD = 3;
+        const INITIAL_IMMUNE = 0;
+
+        const dummyAddress = web3.utils.keccak256('dummy address').slice(-40);
+
+        before('setup', async () => {
+            this.quarantineMock = await QuarantineMock.new(QUARANTINE_PERIOD, INITIAL_IMMUNE);
+            await this.quarantineMock.quarantineContract(dummyAddress);
+        });
+
+        it('should return true from isQuarantined when the contract is quarantined', async () => {
+            expect(await this.quarantineMock.isQuarantined(dummyAddress)).to.be.true;
+        });
+
+        it('should return false from isQuarantined when the quarantine period has passed', async () => {
+            await time.increase((QUARANTINE_PERIOD + 1) * 1000);
+            expect(await this.quarantineMock.isQuarantined(dummyAddress)).to.be.false;
+        });
+
+        it('should revert when attempting to quarantine an empty address', async () => {
```

Comment:
minor: just being nitpicking, does this and the following two tests really follows the description here:
`contract is quarantined` ?